run-docker.sh

```diff
@@ -4,9 +4,13 @@ set -e
 IMAGE_NAME="${IMAGE_NAME:-imap-pipeline-core/imap-mag}"
 
 
+# docker run --rm -it \
```

Comment:
What are the commented out lines for?

Reply:
Oh that is just a tweak of the code that changes the entrypoint to bash which is very handy to debug the container because you then get an interactive shell inside the container rather than something running the entrypoint script. I will make something a bit nicer than a commented out block